editor/src/templates/editor.tsx

```diff
@@ -693,7 +695,7 @@ export const EditorRoot: React.FunctionComponent<{
 }) => {
   return (
     <AtomsDevtools>
-      <JotaiProvider>
+      <JotaiProvider store={jotaiStore}>
```

Comment:
Using an explicit store instead of relying on internal default, so that this can be accessed.